src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts

```diff
@@ -253,7 +254,14 @@ function getAdaptedState(state: PartialState<NavigationState<RootStackParamList>
         }
 
         if (onboardingModalNavigator) {
-            routes.push(onboardingModalNavigator);
+            if (onboardingModalNavigator.state) {
```

Comment:
Could you please add some comments to explain about this change?

Reply:
Done